src/FactorGraph01.jl

```diff
@@ -552,7 +548,7 @@ end
 
 Fetch the variable marginal sample points without the KDE bandwidth parameter.  Use getVertKDE to retrieve the full KDE object.
 """
-function getVal(vA::Vector{DFGVariable}, solveKey::Symbol=:default)::Array{Float64, 2}
+function getVal(vA::Vector{<:DFGVariable}, solveKey::Symbol=:default)::Array{Float64, 2}
```

Comment:
This is because a DFGVariable now takes type parameters for softtype. like DFGFactor

Reply:
sounds good!